extensions/smallrye-reactive-messaging/runtime/pom.xml

```diff
@@ -25,6 +25,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
We should be able to remove the quarkus-vertx-core dependency, as it's a transitive dependency of this one.

Reply:
done, also added dependency between deployments